site/content/en/docs/tasks/setup_sequential_admission.md

```diff
@@ -62,6 +66,28 @@ pods are not all scheduled yet (i.e., the Workload condition remains
 `PodsReady=False`), then the Workload's admission is
 cancelled, the corresponding job is suspended and the Workload is requeued.
 
+### ReQueuing Strategy
+{{% alert title="Warning" color="warning" %}}
+_Available in Kueue v0.6.0 and later_
+{{% /alert %}}
+
+The requeuingStrategy (`waitForPodsReady.requeuingStrategy`) is optional parameters, 
+timestamp (`waitForPodsReady.requeuingStrategt.timestamp`) is defaulting to `Eviction`, and
+backoffLimitCount (`waitForPodsReady.requeuingStrategt.backoffLimitCount`) is defaulting to null.
```

Comment:
remove these lines. It could be explained below.

Reply:
Done.